task/06-conditions-n-loops-tasks.js

```diff
@@ -166,7 +202,7 @@ function isInsideCircle(circle, point) {
  *   'entente' => null
  */
 function findFirstSingleChar(str) {
-    throw new Error('Not implemented');
+    return Array.from(str).filter(s => Array.from(str).filter(s2 => s2 == s).length == 1)[0];
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"